samples/star/src/main/kotlin/com/slack/circuit/star/MainActivity.kt

```diff
@@ -65,4 +87,34 @@ constructor(
   override fun getDefaultViewModelProviderFactory(): ViewModelProvider.Factory {
     return viewModelProviderFactory
   }
+
+  override fun onNewIntent(intent: Intent?) {
+    super.onNewIntent(intent)
+    setContent {
+//      val endString = "see-our-animals/54090287" // animal ID for Veronica.
```

Comment:
I tried testing by running:
`adb shell am start -W -a android.intent.action.VIEW -d "https://socialteesnyc.org/see-our-animals/54090287"`
in the terminal. But just opens to the website instead.